### PR TITLE
cmd: improved help output

### PR DESCRIPTION
**Pull request by i582**

### CODE REVIEW

**Review by quasilyte — COMMENTED** [review]
4 new dependencies to print to the terminal? :cry: 
It also looks like we have 2 color libraries: gookit color and faith color.

**Comment by i582** [comment]
> 4 new dependencies to print to the terminal? 😢
> It also looks like we have 2 color libraries: gookit color and faith color.

Fixed 😅

File: go.sum [hunk 1]
```diff
@@ -1,25 +1,53 @@
 github.com/client9/misspell v0.3.4 h1:ta993UF76GwbvJcIo3Y68y/M3WxlpEHPWIGDkJYwzJI=
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```
Comment (quasilyte) [inline go.sum:3]: I would suggest reverting this file changes.
This log contains unrelated information now.
`go mod tidy` could help/

Comment (i582) [reply go.sum]: Done.

**Comment by quasilyte** [comment]
The code looks good.
Could you provide before/after screenshots or paste the terminal output here?

**Comment by i582** [comment]
> The code looks good.
> Could you provide before/after screenshots or paste the terminal output here?

Before
check_help_before
After
check_help_after

Before
help_before
After
help_after

Before
help_checkers
After
help_checkers_after

**Pull request merged**